.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Getting Started
+
+Thank you for your interest in contributing to the Parse.ly plugin!  We hope this document helps you get set up with everything you need to contribute and we look forward to working with you!
+
+## Reporting Bugs
+
+Please search the [repo's issues](https://github.com/Parsely/wp-parsely/issues) to see if your issue has been reported already and if so, comment on that issue instead of opening a new one.
+
+When creating a new issue, please add specific steps to reproduce the problem, upload any relevant screenshots, and describe what happened and what you expected would happen instead.
+
+## Setting up your environment
+
+### Minimum required versions
```

Comment:
Missing what version of WP the plugin is tested against.

Technically, we only check against the latest version, but we claim to support WP 4.0 (we should add that to the CI matrix).